TCAT/Controllers/RouteOptionsViewController.swift

```diff
@@ -153,13 +153,7 @@ class RouteOptionsViewController: UIViewController {
     // MARK: Route Selection view
 
     private func setupRouteSelection() {
-        // offset for -12 for larger views, get rid of black space
-        routeSelection = RouteSelectionView(frame: CGRect(x: 0, y: -12, width: view.frame.width, height: 150))
-        routeSelection.backgroundColor = Colors.white
-        var newRSFrame = routeSelection.frame
-        newRSFrame.size.height =  routeSelection.lineWidth + routeSelection.searcbarView.frame.height + routeSelection.lineWidth + routeSelection.datepickerButton.frame.height
-        routeSelection.frame = newRSFrame
-
+        routeSelection = RouteSelectionView(frame: CGRect(x: 0, y: 0, width: view.frame.width, height: 130))
         routeSelection.toSearchbar.addTarget(self, action: #selector(self.searchingTo), for: .touchUpInside)
         routeSelection.fromSearchbar.addTarget(self, action: #selector(self.searchingFrom), for: .touchUpInside)
         routeSelection.datepickerButton.addTarget(self, action: #selector(self.showDatePicker), for: .touchUpInside)
```

Comment:
These 3 lines were what I mentioned about button.addTarget being called from the VC. I'll reorganize this in a later diff. Plz don't roast me for this

Reply:
BOOM ROASTED
prob just put this inside a `configure` function